module/mailer/src/Application/DependencyInjection/CompilerPass/MailerStrategyInterfaceCompilerPass.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/**
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE.txt for license details.
+ */
+
+declare(strict_types = 1);
+
+namespace Ergonode\Mailer\Application\DependencyInjection\CompilerPass;
+
+use Ergonode\Mailer\Infrastructure\Sender\MailerSender;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Reference;
+
+/**
+ */
+class MailerStrategyInterfaceCompilerPass implements CompilerPassInterface
```

Comment:
```suggestion
# module/mailer/src/Resources/config/services.yml 
Ergonode\Mailer\Infrastructure\Sender\MailerSender:
        $strategies: !component.notification.mailer_strategy_interface
```
no need for code.

Reply:
I know, but all modules resolve tags with `CompilerPass`. We need to review all and replace it with configuration. But for now `CompilerPass` is the standard. Like i write before all started from SF 4.1, so we need to revalidate it.